docs/reference/dev_help/api_standalone.md

```diff
@@ -17,18 +17,26 @@ catalog = {"Description": "This catalog includes fundamental Python type operato
                         [Bool,    []]]}
 ```
 
-A catalog must be then sourced in Shift either by using the *Catalog Manager* from the Shift UI or by adding the path to the folder containing the catalog file to the `SHIFT_CATALOG_PATH` environment variable.
+>[!NOTE]
+> The *Operators* list from the `catalog` is a list of lists containing two elements. The first one corresponds to the operator class, and the second one is a list of compatible hosts for the operator. For example, if the second element is `["maya", "houdini"]`the operator will be only compatible with those DCC's and work under those contexts. Refer to [Integrations & Resources](../../integration_resources/integrations_resources.md) for more information.
+
+There are two ways of adding a custom catalog to Shift:
+
+- **As a User Catalog**: Use the [Catalog Manager](../catalogs/#the-catalog-manager) to source the catalog file and add it to Shift. This catalog is saved in the user preferences.
+- **As a Environment Catalog**: Add the **path to the directory** the containing the catalog file to the `SHIFT_CATALOG_PATH` environment variable. All files in this path containing a `catalog` object will be identified as custom catalogs and loaded by Shift.
 
 ## Operator Syntax
 
-Inside the catalog file you can create your custom operators as Python classes. Here is an example of its syntax:
+All Shift operators inherit from the `SOperator` Shift's Python class. To write a custom operator, the `__init__` and `execute` methods must be overwritten. 
+
+Here is an example of an operator class syntax:
 
 ```python
 from shift.core.workflow import SOperator
 from shift.core.workflow import SPlug
 from shift.core.constants import SType
 from shift.core.constants import SDirection
-
+ 
 class MyOperator(SOperator):
 
```

Comment:
I would add a docstrings flagging that the docstring of the class will be use like Help test for the operator inside Shift:
```suggestion
    """The docstring provided in the Operator class will be use like the operator help text inside Shift. You can provide a detailed description of the Operator purpose and the requirements for the inputs and outputs here."""
```

Reply:
Hum, thinking on what I would like like an external developer I would like to have it in the example. But If you prefer to just add a note saying that the docstring of the Operator class will became the documentation of the Operator in Shift, It´s ok too.

Reply:
I added the informaiton in the docstring